src/components/block.ts

```diff
@@ -446,7 +446,7 @@ export default class Block {
    * @return {BlockTune[]}
    */
   public makeTunes(): BlockTune[] {
-    const tunesList = [MoveUpTune, DeleteTune, MoveDownTune];
+    const tunesList = [];
```

Comment:
These three tunes are default Block settings. Your line is removing default behavior